e2e/playwright.config.ts

```diff
@@ -0,0 +1,78 @@
+import { defineConfig } from '@playwright/test';
+// import { AzureReporterOptions } from '@alex_neo/playwright-azure-reporter/dist/playwright-azure-reporter';
+import dotenv from 'dotenv';
+import path from 'path';
+
+const envPath = path.resolve(__dirname, '../.env');
+dotenv.config({ path: envPath });
+
+export default defineConfig({
+  testDir: './tests',
+  snapshotPathTemplate: '{testDir}/__screenshots__/{testFilePath}/{arg}{ext}',
+  /* Run tests in files in parallel */
+  fullyParallel: false,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: 1,
+  /* Opt out of parallel tests on CI. */
+  reporter: [
+    ['list'],
+    // [
```

Comment:
can we remove this (and check for more?) commented code?

Reply:
So as I mentioned the commented out code is the code used for reporting the tests status to Azure or any other Test management tool. For this reason I would like to keep it and use it uncommented when needed. For most part it will stay commented since we don't want a report to be sent per each commit to PR.